app/assets/stylesheets/common/admin/schema_theme_setting_editor.scss

```diff
@@ -1,156 +1,158 @@
 .schema-theme-setting-editor {
```

Comment:
This file looks complicated but all I did is just add `.schema-theme-setting-editor__wrapper` and a whole bunch of stuff got indented.